src/App.jsx

```diff
@@ -1,104 +1,31 @@
 import './App.scss';
+import postsFromServer from './api/posts.json';
+import usersFromServer from './api/users.json';
```

Comment:
The import statement for `usersFromServer` is redundant as it is already imported on line 3. Consider removing the commented-out import on line 8 to clean up the code.